src/validate.test.ts

```diff
@@ -0,0 +1,96 @@
+import { check } from '@superfaceai/cli/dist/logic/check';
+import { SuperJson } from '@superfaceai/one-sdk';
+
+import * as util from './util';
+import * as validate from './validate';
+
+jest.mock('@superfaceai/cli/dist/logic/check');
+jest.mock('./util');
+
+describe('validate', () => {
+  beforeEach(() => {
+    (util.loadSuperJson as jest.Mock).mockReturnValue(new SuperJson({}));
```

Comment:
Nitpick: you could probably use `mocked` from ts-jest/utils to get rid of casting to jest.Mock

`mocked(loadSuperJson).mockReturnValue`

Reply:
🙇 didn't know it.